Readability.js

```diff
@@ -1678,15 +1678,18 @@ Readability.prototype = {
    *
    * @return boolean Whether or not we suspect parse() will suceeed at returning an article object.
    */
-  isProbablyReaderable: function() {
-    var nodes = this._doc.getElementsByTagName("p");
-    if (nodes.length < 5) {
-      return false;
-    }
-
-    var possibleParagraphs = 0;
+  isProbablyReaderable: function(helperIsVisible) {
+    var nodes;
+    if (typeof this._doc.querySelectorAll != "undefined")
```

Comment:
I don't understand how you would do that - `_concatNodeList` takes two node lists; at that point it's too late for `querySelectorAll`...